### PR TITLE
AR : fix contribute.md 

### PR DESCRIPTION
**Pull request by 3imed-jaberi**

### CODE REVIEW

**Comment by ghost** [comment]
@nodejs/nodejs-ar , Please have a review, thanks!

File: locale/ar/get-involved/contribute.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ layout: contribute.hbs
...
 ## الإبلاغ عن مشكلة
 
-إذا وجدت أي شيء تعتقد انه مشكلة في الـ Node.js، لا تتردد في في فتح مشكلة على المشروع في الـ GitHub. عند طرح للمشكلة، احرص على ان تعبر عنها بفحوصات قابلة للاعادة، كما يجب على تلك الفحوصات ألا تستعين بأي مكتبات خارجية. بشكل أوضح، يجب على تلك الفحوصات ان تكون قابلة للتنفيذ اعتماداً على الـ Node.js لا أكثر.
+إذا وجدت أي شيء تعتقد انه مشكلة في الـ Node.js، لا تتردد في في فتح مشكلة على المشروع في الـ GitHub. عند طرح مشكلتك، احرص على ان تعبر عنها بفحوصات قابلة للاعادة، كما يجب على تلك الفحوصات ألا تستعين بأي مكتبات خارجية. بشكل أوضح، يجب على تلك الفحوصات ان تكون قابلة للتنفيذ اعتماداً على الـ Node.js لا أكثر.
 
 عند التبليغ عن مشكل، نحتاج اكبر قدر ممكن من المعلومات حول بيئة التشغيل  التي تستعملها. لا ندري أي معلومة لها صلة وثيقة بالمشكلة عند محاولة حصر المشكلة، قم بتضمين المعلومات التالية على الأقل:
 
 * إصدار الـ Node.js
-* منصة التشغيل التي تعمل عليها (macOS, SunOS, Linux, Windows)
+* منصة التشغيل التي تعمل عليها (macOS, SmartOS, Linux, Windows)
```
Comment (ram-you) [inline locale/ar/get-involved/contribute.md:21]: Why change SunOS to SmartOs ?

Comment (3imed-jaberi) [reply locale/ar/get-involved/contribute.md:21]: https://nodejs.org/en/get-involved/contribute/ 

image

Comment (ram-you) [reply locale/ar/get-involved/contribute.md:21]: Sorry, I had not seen the original (english version).

File: locale/ar/get-involved/contribute.md [hunk 2]
```diff
@@ -30,14 +30,14 @@ layout: contribute.hbs
...
+إذا كنت تتسائل حول كيفية البدء، يمكنك الإطلاع على [Node Todo](https://www.nodetodo.org/)، حيث يمكن له أن يريك كيفية القيام بمساهمتك الأولى.
 
 ## كيف تصبح مساهماً
 
-عندما تصبح مساهما، تمتلك تأثيراً أكبر على المشروع. يمكن للمساهمين مساعدة بعضهم عبر مراجعة مساهماتهم، و تقصي المشكلات، و حتى تشكيل مستقبل المشروع، كما يمكن للأفراد الذين تحددهم لجنة التوجيه التقني بأنهم يقدمون مساهمات نوعية أن يتم ترقيتهم لمتعاونين، و تم منحهم حق الكتابة في المشروع، و تمثل الأنشطة التي يتم أخذها بعين الاعتبار (و غير محدودة بها) نوعية:
+عندما تصبح مساهما، تمتلك تأثيراً أكبر على المشروع. يمكن للمساهمين مساعدة بعضهم عبر مراجعة مساهماتهم، و تقصي المشكلات، و حتى تشكيل مستقبل المشروع، كما يمكن للأفراد الذين تحددهم لجنة التوجيه التقني بأنهم يقدمون مساهمات نوعية أن يتم ترقيتهم لمتعاونين، و يتم منحهم حق الكتابة في المشروع، حيث تمثل الأنشطة التي يتم أخذها بعين الاعتبار (ولكنها غير محدودة بها) نوعية:
 
-* الكتابة في الشفرة المصدرية و طلبات السحب
-* الكتابة في التوثيق و طلبات السحب
+* الإلتزامات في الشفرة المصدرية و طلبات السحب
```
Comment (Aissaoui-Ahmed) [inline locale/ar/get-involved/contribute.md]:
```suggestion
* المساهمة في الشفرة المصدرية و طلبات السحب
```

**Review by celyes — APPROVED** [review]

File: locale/ar/get-involved/contribute.md [hunk 3]
```diff
@@ -30,14 +30,14 @@ layout: contribute.hbs
...
+* الإلتزامات في التوثيق و طلبات السحب
```
Comment (Aissaoui-Ahmed) [inline locale/ar/get-involved/contribute.md]:
```suggestion
* المساهمة في التوثيق و طلبات السحب
```

**Review by Aissaoui-Ahmed — CHANGES_REQUESTED** [review]

**Review by Aissaoui-Ahmed — APPROVED** [review]
Great 💯

**Pull request merged**